### DISCUSSION TITLE
User discovers this one cool trick for mixed composition with ControlNet

### DISCUSSION THREAD

**Discussion by Njasa2k** [lllyasviel/ControlNet#100] [Show and tell]
Redditor u/Ne_Nel discovers this mixed composition trick with ControlNet and img2img using Automatic1111.
![mycpob07guia1](https://user-images.githubusercontent.com/99513716/219842785-d13bacee-ab37-4274-a44c-f6f68b569c64.png)
![isdd1x17guia1](https://user-images.githubusercontent.com/99513716/219842801-806fa4f1-1c6f-40bf-b028-bf3a7f57b188.png)
![7burkk07guia1](https://user-images.githubusercontent.com/99513716/219842803-5d52774b-ec11-42b7-9e70-e0ad9e83012c.png)
![qrrjzj07guia1](https://user-images.githubusercontent.com/99513716/219842804-32644a1e-5b2c-46ce-9289-443e19172a12.png)
![1g7vdr17guia1](https://user-images.githubusercontent.com/99513716/219842805-06b8cc0a-9466-4e86-b429-27344516e6d1.png)
![7xqi8i07guia1](https://user-images.githubusercontent.com/99513716/219842806-c235b057-62dc-4d1e-9fb0-e4163d0ce79c.png)
![dxmzre07guia1](https://user-images.githubusercontent.com/99513716/219842807-555a3792-6500-4b50-af4f-06b9dba7b9b0.png)
![og2uoh07guia1](https://user-images.githubusercontent.com/99513716/219842809-1e164f40-e36c-48c6-a0a4-4998d7042e27.png)

**Comment by Njasa2k** [comment]
More images by u/Ne_Nel
![34etc4inrxia1](https://user-images.githubusercontent.com/99513716/219872409-8cb9fc6e-aab0-410c-9ff4-74b6e8e7254d.png)
![4fxl78inrxia1](https://user-images.githubusercontent.com/99513716/219872411-018ce699-4b5f-43ac-b956-667ee01a4fae.png)
![vhzr64inrxia1](https://user-images.githubusercontent.com/99513716/219872413-58417fbf-0e50-44a6-b89f-63239aaf0030.png)
![t5sy35inrxia1](https://user-images.githubusercontent.com/99513716/219872417-72057ba6-8eb9-4c3f-aa8e-a5c632690e82.png)
![l9nogxhnrxia1](https://user-images.githubusercontent.com/99513716/219872418-62831864-3172-4409-877d-8de7c33851cc.png)
![t6x3bfjnrxia1](https://user-images.githubusercontent.com/99513716/219872419-33ffd5c0-c0c4-412a-a091-80b7f8549607.png)
![6ymp9grqrxia1](https://user-images.githubusercontent.com/99513716/219872420-99f82419-0ba1-4d80-aa9c-0241915e71b0.png)

**Comment by Njasa2k** [comment]
u/Ne_Nel discovers how to manipulate lighting
![v0y07soi60ja1](https://user-images.githubusercontent.com/99513716/219892937-50ce1d38-9d01-4ac6-b7c8-b42c4cc6e31f.png)
![osr8nroi60ja1](https://user-images.githubusercontent.com/99513716/219892940-c9abf0d2-800e-4634-b3f3-1e1a1a41a18e.png)
![jsrkmvoi60ja1](https://user-images.githubusercontent.com/99513716/219892941-f45625fe-db56-49c2-8840-ff3206c5c4dc.png)
![7jqpjuoi60ja1](https://user-images.githubusercontent.com/99513716/219892942-60f478f7-1793-454c-bc4e-66b8a1ec73ef.png)
![z9bldmoi60ja1](https://user-images.githubusercontent.com/99513716/219892943-584ea903-9cd6-47f6-9e80-6a8e1f1c03a9.png)

**Comment by FurkanGozukara** [comment]
@ninjasaid2k could you give more information how to do?

step by step
